tchannel-core/src/main/java/com/uber/tchannel/api/TChannel.java

```diff
@@ -84,7 +84,6 @@
     private final int port;
     private String listeningHost = "0.0.0.0";
     private int listeningPort;
-    private ExecutorService exectorService;
```

Comment:
I am not sure that this has to go, but since it is not used having it private is not wise. Perhaps protected.

Reply:
this member was never used - while creating an impression it is somehow related to how the class works.